### PR TITLE
159965445 Add pagination support for articles

### PR DESCRIPTION
**Pull request by Quantum-35**
## What does this PR do?
- Adds Pagination Support to article Landing Page.

## Description of Task to be completed?
- The user can easily navigate to previous articles using pagination.

## How should this be manually tested?
Clone the repo, cd into ah-leagueoflegends-frontend. Run **npm install** or **yarn add**. Then run yarn start.
- Start your Django backend server.
- Open the home page on your browser.
- Login
- Navigate to the home page and scroll down to navigate to previous articles.

## What are the relevant pivotal tracker stories?
[#159965445](https://www.pivotaltracker.com/story/show/159965445)

### Landing Page with pagination support.
screen shot 2018-10-26 at 12 20 20

### CODE REVIEW

**Review by Derrickkip — APPROVED** [review]

**Review by Teddykavooh — APPROVED** [review]

File: src/containers/Articles/Read/index.js [hunk 1]
```diff
@@ -2,20 +2,47 @@ import React, { Component } from 'react';
...
+    perPage: 10,
+    pageCount: 1,
+  }
+
   componentDidMount() {
     const { getAllArticles } = this.props;
-    getAllArticles();
+    const { perPage, offset } = this.state;
+    getAllArticles(perPage, offset);
+    getAllArticles(perPage, offset);
```
Comment (kevintumbo) [inline src/containers/Articles/Read/index.js]: duplication function calls

File: src/containers/Articles/Read/index.js [hunk 2]
```diff
@@ -2,20 +2,47 @@ import React, { Component } from 'react';
...
+    axios.get(`${BACKEND_URL}api/articles`)
+      .then((res) => {
+        const pageCount = Math.ceil(res.data.articles.count / res.data.articles.results.length);
+        this.setState({ pageCount });
+      })
+      // eslint-disable-next-line
+      .catch(err => console.log(err));
```
Comment (kevintumbo) [inline src/containers/Articles/Read/index.js]: remove console.log

**Review by kevintumbo — CHANGES_REQUESTED** [review]

File: src/containers/Articles/Read/index.js (hunk 1 above)
Comment (Quantum-35) [reply src/containers/Articles/Read/index.js]: On it.

File: src/containers/Articles/Read/index.js (hunk 2 above)
Comment (Quantum-35) [reply src/containers/Articles/Read/index.js]: On it.

**Review by kevintumbo — APPROVED** [review]

**Pull request merged**